testing/jupyterlab/test-info-pages/no-tab-trap-initial-page.md

```diff
@@ -0,0 +1,38 @@
+# No Tab Trap on Initial Page
+
+- id: 1wlgh6
+
+## Description
+
+When a user loads the app for the first time, they should be able to use the tab
+key to navigate without getting trapped.
+
+## Applicability
+
+Which apps does this test apply to.
+
+- JupyterLab
+
+## Accessibility Criteria
+
+This test is related to:
+
+- [WCAG Success Criterion 2.1.2 No Keyboard
+  Trap](https://www.w3.org/TR/WCAG22/#no-keyboard-trap)
+
+If this test fails, it means the app fails to conform to 2.1.2.
+
+If this test passes, however, it does not mean that the app conforms to 2.1.2.
+It only means that the app does not violate 2.1.2 for this specific test. This
+test can be combined with others to increase confidence that the app as a whole
```

Comment:
```suggestion
test can be combined with tab trap tests for other pages or page states to increase confidence that the app as a whole
```

If the intent is to have this as a general disclaimer, then I think what you have works. If you want to give an idea on this test specifically, I think it would be nice to get a hint of why a passing test does not equate to conformance and/or what would be more thorough.

I'm trying to think of this as if I'm someone without a lot of past web accessibility experience, and I could imagine myself being like "well, what does the test do if it doesn't tell me if I am complying with the standard?" So I imagine pointing out the "why" (like how this only tests one page of a whole app) could help.

Reply:
Oh yes, I like this suggestion. We can have some general abstract language somewhere else but in the test info page we should make it as concrete as possible.

Reply:
aha Isabela added on my thoughts!

What I want to know is: what does this test tell me? what have I learned?

Reply:
👍 love this, will reword